src/FakeItEasy/Creation/DummyValueResolver.cs

```diff
@@ -26,6 +26,7 @@ public DummyValueResolver(DynamicDummyFactory dummyFactory, IFakeObjectCreator f
             this.strategyCache = new ConcurrentDictionary<Type, ResolveStrategy>();
             this.strategies = new ResolveStrategy[]
                 {
+                    new ResolveVoidByRetuningNullStrategy(),

```

Comment:
Late-breaking demand:

```suggestion
                    new ResolveVoidByReturningNullStrategy(),

```

It's not a piano.

Reply:
Oops! Good catch. I'll amend the commit to avoid another round of approve/squash/force push/approve

> It's not a piano.

🤣